### PR TITLE
Release 7.6.0

### PR DESCRIPTION
**Pull request by jyaganeh**
## Version 7.6.0 - July 3, 2024
Minor release that updates the Android SDK to 18.1.1 and the iOS SDK to 18.5.0.

### Changes
- Updated Android SDK to 18.1.1
- Updated iOS SDK to 18.5.0
- Updated airship-mobile-framework-proxy to 7.0.0
- Updated Kotlin version to 1.9.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +17,7 @@ Minor release that updates iOS SDK to 18.4.1, updates Android compileSDKVersion
 - Updated airship-mobile-framework-proxy to 6.3.1
 - Fixed Event Emitter bug
 - Updated Android compileSDKVersion from 33 to 34 and set source and target compatibility to Java 17
-- Updated Android example build configration
+- Updated Android example build configuration
```
Comment (crow) [inline CHANGELOG.md:20]: Nice

**Review by crow — APPROVED** [review]

File: android/build.gradle [hunk 2]
```diff
@@ -60,7 +61,6 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutine_version"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutine_version"
     implementation "com.urbanairship.android:urbanairship-fcm:$airship_version"
```
Comment (rlepinski) [inline android/build.gradle:66]: probably dont need this?

**Comment by rlepinski** [comment]
can we add the ios log level privacy setting? You just have to pass it through on the dart side

File: example/lib/main.dart [hunk 3]
```diff
@@ -28,9 +28,11 @@ void main() {
   ]);
 
   var config = AirshipConfig(
-      defaultEnvironment: ConfigEnvironment(
-          appKey: "APP_KEY",
-          appSecret: "APP_SECRET"));
+    defaultEnvironment: ConfigEnvironment(
+        appKey: "APP_KEY",
+        appSecret: "APP_SECRET",
+        ios: IOSEnvironment(logPrivacyLevel: LogPrivacyLevel.public)),
```
Comment (crow) [inline example/lib/main.dart:34]: AirshipLogPrivacyLevel?

**Pull request merged**